cognee/infrastructure/databases/hybrid/falkordb/FalkorDBAdapter.py

```
@@ -0,0 +1,267 @@
+    async def create_vector_index(self, index_name: str, index_property_name: str):
+        graph = self.driver.select_graph(self.graph_name)
+
+        if not self.has_vector_index(graph, index_name, index_property_name):
+            graph.create_node_vector_index(index_name, index_property_name, dim = self.embedding_engine.get_vector_size())
+
+    def has_vector_index(self, graph, index_name: str, index_property_name: str) -> bool:
+        try:
+            indices = graph.list_indices()
+
+            return any([(index[0] == index_name and index_property_name in index[1]) for index in indices.result_set])
+        except:
```

Ruff

Do not use bare `except`

(E722)